### PR TITLE
Windows support cleanup tasks

### PR DESCRIPTION
**Pull request by jnstahl**

### CODE REVIEW

**Comment by Nimrodda** [comment]
Thanks for the PR. Could you please rebase on latest origin/master?

**Comment by jnstahl** [comment]
Ok this should be fixed

File: pubspec.yaml [hunk 1]
```diff
@@ -28,3 +28,6 @@ flutter:
         pluginClass: FlutterNsdPlugin
       macos:
         pluginClass: SwiftFlutterNsdPlugin
+      windows:
+        fileName: flutter_nsd_plugin.cpp
+        pluginClass: FlutterNsdPlugin
```
Comment (Nimrodda) [inline pubspec.yaml]: We already have this in lines 26-28

File: windows/mdns_impl.c [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
-#ifdef _WIN32
+
 #include <winsock2.h>
 #include <iphlpapi.h>
 #define sleep(x) Sleep(x * 1000)
-#else
-#include <netdb.h>
-#include <ifaddrs.h>
+
+#if (_DEBUG == 0)
```
Comment (Nimrodda) [inline windows/mdns_impl.c:10]: Could you please add instructions in the README.md under Windows section how can one enable this?

**Review by Nimrodda — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (jnstahl) [reply pubspec.yaml]: Ugh. This came back during rebase

File: windows/mdns_impl.c (hunk 2 above)
Comment (jnstahl) [reply windows/mdns_impl.c:10]: Actually the flutter toolchain automatically sets this for debug builds.

**Review by Nimrodda — APPROVED** [review]
Thanks!

**Pull request merged**